BeagleBone_black/src/communications/communications.cpp

```diff
@@ -89,7 +92,9 @@ int Communications::receiveRunLength()
 
 int Communications::receiveMessage()
 {
+  int n_;
   n_ = read(sockfd_, buffer_, 255);
```

Comment:
could be `int n_ = read(sockfd_, buffer_, 255);`

Reply:
👍 Except it should be `n` rather than `n_`